app/gateway/changelog.md

```diff
@@ -3182,6 +3180,80 @@ openid-connect
 * Bumped `lodash` for Kong Manager from 4.17.15 to 4.17.21
 
 
+## 2.8.4.5
+**Release Date** 2023/11/28
+
+### Features
+#### Core
+* Added support for observing the time consumed by some components in the given request.
+* A unique Request ID is now populated in the error log, access log, error templates, log serializer, and in a new X-Kong-Request-Id header (configurable for upstream/downstream using the `headers` and `headers_upstream` configuration options).
```

Comment:
```suggestion
* Added a unique Request ID that is now populated in the error log, access log, error templates, log serializer, and a new `X-Kong-Request-Id` header. 
  This configuration can be customized for upstreams and downstreams using the 
  [`headers`](/gateway/2.8.x/reference/configuration/#headers) and 
  [`headers_upstream`](/gateway/2.8.x/reference/configuration/#headers_upstream) configuration options. 
  [#11663](https://github.com/Kong/kong/pull/11663)
```
entry from 3.5. this is a feature that's being backported.

looks like we need to backport these config changes too.